### PR TITLE
Microverse Mobile Portfolio Setup

### PR DESCRIPTION
**Pull request by peterboro**
In this pull request, I implemented the following elements:

- Setup the HTML & CSS linters.
- Built the first 2 sections using the Figma template.
- Modified some changes to my HTML and CSS code.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.social_links {
+  margin-left: -57px;
+}
```
Comment (zieeco) [inline style.css:90]: - Please, kindly adjust your styling to match the Figma design below:

*your design*
designn

*figma*



figmaa

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @peterboro ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
_Check the comments under the review._

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    <div class="container-1">
+        <div class="font">
+            <header class="header">
+                <a class="myLogo"> My Logo </a>
+                <nav class="burger">
+                    <ul class="ul-image">
+                        <li>
+                            <a href="#">
+                <img src="/Images/Union.png" alt="union-background"/>
+            </a>
```
Comment (Reem-lab) [inline index.html:23]: - [ ] Great job so far on working on the styles but as the previous code reviewer asked to adjust the style so I can see here that the logo and the Hamburger menu are not aligned with the Figma design so please Fix the aligning and use list-style: none to delete the bullet points.

   - Your design :
       
image

   - Figma design 
 
image

File: README.md [hunk 3]
```diff
@@ -1,2 +1,54 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+- GitHub: [@peterboro]
```
Comment (Reem-lab) [inline README.md:54]: - [ ] I think here you didn't update the MIT file as the previous code reviewer asked so please add it to your project you can see the template here and customize it to your information [MIT](https://github.com/Reem-lab/MyPortfolio/blob/main/LICENSE).👌🏻

**Review by Reem-lab — CHANGES_REQUESTED** [review]
## Hi @peterboro , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
 - Good GitHub flow ✅✅
 -  No linters error ✅✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (danmainah) [reply index.html:23]: - [ ] [optional] @peterboro great work implementing changes requested, you could also improve in your design check the comparison below some background images are not well positioned.

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image | image

**Review by danmainah — APPROVED** [review]
Hi @peterboro , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good PR and Description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**